src/bindings/c/src/ov_property.cpp

```diff
@@ -16,6 +16,7 @@ const char* ov_property_key_device_capabilities = "OPTIMIZATION_CAPABILITIES";
 const char* ov_property_key_model_name = "NETWORK_NAME";
 const char* ov_property_key_optimal_batch_size = "OPTIMAL_BATCH_SIZE";
 const char* ov_property_key_max_batch_size = "MAX_BATCH_SIZE";
+const char* ov_property_key_config_file = "CONFIG_FILE";
```

Comment:
```suggestion
const char* ov_property_key_intel_gpu_config_file = "CONFIG_FILE";
```
Move gpu's property to `src/bindings/c/include/openvino/c/gpu/gpu_plugin_properties.h`

Reply:
Ok, I moved the declaration from `ov_property.h` to `gpu_plugin_properties.h` and also renamed. I haven't moved the definition from `ov_property.cpp`.